packages/astro/src/default/layouts/Layout.astro

```diff
@@ -24,16 +24,35 @@ const baseURL = import.meta.env.BASE_URL;
     <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap" rel="stylesheet" />
     <link href="https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@400;700&display=swap" rel="stylesheet" />
     <ViewTransitions />
+    <script is:inline>
+      setTutorialKitTheme();
```

Comment:
Calling this function as an inline script guarantee that we always add the theme attribute before any dom as been rendered which makes sure no transition animations are rendered.

Without this change, on dark mode a transition animation was visible on every page reload.

Reply:
Seems to work nice! 👍